dhcpv4/modifiers.go

```diff
@@ -43,6 +43,18 @@ func WithGatewayIP(ip net.IP) Modifier {
 	}
 }
 
+//WithRelayOptions copies the relay options from the request to the reply
```

Comment:
super nit: Add a space at the beginning and a period at the end `// WithRelay...`